CodeBenders/Assets/Scripts/ProjectileGenerator.cs

```diff
@@ -79,13 +79,15 @@ public class ProjectileGenerator : MonoBehaviour
         float gravityScale)
     {
         // include a GUID in the name so no name collision could be possible
-        var projectile = new GameObject($"Projectile ({Guid.NewGuid()})", 
+        var projectile = new GameObject($"Projectile ({Guid.NewGuid()})",
             new []{typeof(CircleCollider2D), typeof(SpriteRenderer), typeof(Rigidbody2D)});
         // tag the projectile
         projectile.tag = projectileTag;
+        //scale the projectile
+        projectile.transform.localScale = new Vector3(2, 2, 2);
```

Comment:
Works well for a hotfix. But after midterm I would still suggest making the scale factor a proper parameter (see 19e362e27681469bfb4834b32b800e88a9188553 for that).